docs/source/manual/correlation.rst

```diff
@@ -0,0 +1,59 @@
+.. _beadledom-correlation:
+
+beadledom-correlation
+======================
+
+This project encapsulates and provides access to the Correlation ID of the current transaction. A Correlation ID is a unique identifier that can be used to connect (correlate) calls through multiple services that are part of a single transaction. This project provides access to the current Correlation Id and allows frameworks to control the lifecycle of the Correlation Id.
+ 
+Download
+--------
+
+Download using Maven
+~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: xml
+
+  <dependencies>
+      ...
+      <dependency>
+          <groupId>com.cerner.beadledom</groupId>
+          <artifactId>beadledom-correlation</artifactId>
+          <version>[Insert latest version]</version>
+      </dependency>
+      ...
+  </dependencies>
+
+Usage
+-----
+
+Setting the Correlation ID
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: java
+  CorrelationContext correlationContext = new ThreadLocalCorrelationContext();
```

Comment:
These need to be updated with your latest change.

Reply:
Updated https://github.com/cerner/beadledom/pull/180/commits/fef50c057da3fd5605b5312ab6c1d4774e57cd09